### PR TITLE
docs: new content and styling

### PR DESCRIPTION
**Pull request by NickIliev**
Restructure the main drawer, added Get Started articles and some styling improvements

### CODE REVIEW

File: user-guide/settings/connections.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ __Fiddler listens on port:__ textbox controls which port Fiddler Everywhere uses
 
 ## Act as a System Proxy on Startup
```
Comment (stoskov) [inline user-guide/settings/connections.md:21]: I guess Connections and Gateway sections are still to be updated with fewer images?

File: user-guide/live-traffic/inspector-types.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+---
+title: Inspector
```
Comment (stoskov) [inline user-guide/live-traffic/inspector-types.md:2]: Make it plural - Inspectors

File: get-started/create-api-request.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    >tip Use the **Raw** view to inspect in detail the request that will be sent to the server. The view is read-only.
+
+3. Click on the **Execute** button to send the composed API request.
+
+4. Inspect the result in the Composer's **Response inspector**. Like the inspectors for the **Live Traffic** tab, the **Response** inspector for the Composer provides multiple views (for example, *Headers*, *Raw*, *JSON*, etc.).
+
+5. (**_Optional_**) You can save the request by clicking on the **Save** button (to the right of **Execute**). Saved requests are loaded in a collection in the **Requests list** and can be reused or shared with collaborators.
+
+![Creating API request](../images/composer/create-api-request.png)
```
Comment (stoskov) [inline get-started/create-api-request.md:29]: Add a link to the full Composer description in User Guide

File: user-guide/composer.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+## Composer
+
+The __Composer__ tab enables you to manually build and send HTTP, HTTPS, and FTP requests.
+
+## Get Started
+
+The __Composer__ contains three major sections used to construct a request and observe the response.
+
+- The top section contains a drop-down for selecting __HTTP Methods__, an __URL field__, a drop-down to select the used __HTTP version__, and an __Execute__ button.
```
Comment (stoskov) [inline user-guide/composer.md:17]: Make these simple paragraphs instead of a dash list

File: user-guide/composer.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+![HTTP Versions drop-down](../images/composer/composer-http-version.png)
+
+## Request Composer
+
+The __Request Composer__ is allowing you to further modify your request by adding your custom __Headers__, __Params__, __Body__, and observing the composed request via the __Raw__ view.
+
+### Headers
+
+Enables you to add/modify your request headers (e.g., `Content-Type`, `Authorization`, etc.). 
+ - Add a new header by entering the header key-value pair and then clicking the tick.
```
Comment (stoskov) [inline user-guide/composer.md:63]: Make these paragraphs and not list

File: user-guide/composer.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+ - Adds the new header to your request. The header can now be disabled/enabled or completely deleted.
+  ![Added header](../images/composer/composer-headers-after.png)
+
+>important Most of the servers using newer versions of TLS will require a **User-Agent** header to be set. By default, Fiddler Everywhere will a **User-Agent** key set with value **Fiddler Everywhere** and description **_Lets servers and network peers identify the application, operating system, vendor, and version of the requesting user agent_**. The header is non-mandatory, but keep in mind that without valid **User-Agent**, some requests to secure servers might fail.
+![Default User-Agent](../images/composer/composer-user-agent.png)
+
+### Params
+
+Enables you to add query parameters to your request URL easily. Adds any key-value pair appended through the  __Params__ view to the request URL.
+ - Add new query params by entering the params key-value pair and then clicking the tick.
```
Comment (stoskov) [inline user-guide/composer.md:74]: Make these paraghraphs

File: user-guide/composer.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+Enables you to add/modify your request headers (e.g., `Content-Type`, `Authorization`, etc.). 
+ - Add a new header by entering the header key-value pair and then clicking the tick.
+ ![Adding header](../images/composer/composer-headers-before.png)
+ - Adds the new header to your request. The header can now be disabled/enabled or completely deleted.
+  ![Added header](../images/composer/composer-headers-after.png)
+
+>important Most of the servers using newer versions of TLS will require a **User-Agent** header to be set. By default, Fiddler Everywhere will a **User-Agent** key set with value **Fiddler Everywhere** and description **_Lets servers and network peers identify the application, operating system, vendor, and version of the requesting user agent_**. The header is non-mandatory, but keep in mind that without valid **User-Agent**, some requests to secure servers might fail.
+![Default User-Agent](../images/composer/composer-user-agent.png)
+
+### Params
```
Comment (stoskov) [inline user-guide/composer.md:71]: Shouldn't these be H4?

File: user-guide/live-traffic/live-traffic.md [hunk 8]
```diff
@@ -0,0 +1,244 @@
...
+
+## Live Traffic Toolbar
+
+The Live Traffic toolbar provides various functionalities like streaming mode, decoding content, filtering by criteria, saving, and sharing sessions.
+![Live Traffic Toolbar](../../images/livetraffic/websessions/websessions-toolbar-all.png)
+
+To start capturing, you need to enable __Live Traffic__ switch to **Capturing** mode. Use the off switch state to **pause** the traffic capturing.
+![Enabling Live Traffic](../../images/livetraffic/websessions/websessions-live-traffic-capturing.png)
+
+### Stream
```
Comment (stoskov) [inline user-guide/live-traffic/live-traffic.md:25]: Shouldn't these be H4?

File: user-guide/live-traffic/autoresponder.md [hunk 9]
```diff
@@ -0,0 +1,202 @@
+---
```
Comment (stoskov) [inline user-guide/live-traffic/autoresponder.md:1]: I guess this is yet to be revised?

File: user-guide/settings/connections.md (hunk 1 above)
Comment (NickIliev) [reply user-guide/settings/connections.md:21]: Removed obsolete images and introduced the new styling with H4

**Pull request merged**